circle.yml

```diff
@@ -49,6 +49,9 @@ commands:
           working_directory: /tmp
           command: |
             if [ ! -f testfloat_gen ]; then
+              # GCC compiler is expected, create link from clang if gcc not available.
+              type gxx || (cd $(dirname $(which clang)) && ln -s clang gcc)
```

Comment:
It uses `gxx`? I only ever had `gcc` and `g++` and never `gxx`.

Reply:
typo